### ISSUE TITLE
Solar Panels?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#20]
## Final tasks as of Dec-2019
+ [x] Fixing snags https://github.com/dwyl/home/issues/20#issuecomment-549135213
+ [x] Final EDP paperwork [dependency on #70 being completed]
+ [x] Get application hooked up for solar panels [dependency on having the internet working]
+ [ ] DAPE to finish configuration of equipment remotely

# _Context_

**`home`** is based in a city centre with _good_ infrastructure. 🌇  https://en.wikipedia.org/wiki/Braga
We have access to the National Grid which provides a reliable/consistent supply.
We don't _need_ to tie up capital on Solar Panels and storage (batteries) up-front 
because there is no _shortage_ of electricity in the Portugal - many other countries have regular/prolonged power cuts ... e.g Eskom's "planned blackouts" in South Africa: 🌍🔌🚫😕
https://www.bbc.com/news/av/world-africa-31126033/south-africa-faces-power-crisis-and-blackouts

# Portugal Renewable Energy _Facts_ 🌬 

Investment in renewable energy, _specifically_ **wind** has been relentless! 
In the 10 years from 2003 to 2013 wind generation capacity grew from 0.3GW to 4.73GW; **16x**!
![portugal-investment-renewable-energy-2000-2016](https://user-images.githubusercontent.com/194400/47279052-d98d7480-d5c5-11e8-9840-49baf3c54ca2.png)

But the generation _capacity_ does not even tell the "full story", the (_annual_) electricity _generated_ by wind is even more _impressive_: from 0.48TWh to 12,19TWh; **25x**!!

![portugal-wind-2003-0 48twh](https://user-images.githubusercontent.com/194400/47280732-b23ba500-d5cf-11e8-8d03-6e2a6c803c2d.png)
![portugal-wind-2016-12twh](https://user-images.githubusercontent.com/194400/47280736-b49dff00-d5cf-11e8-878a-8b0e717dfffc.png)

We will discuss the "grey area" (_fossil fuels_) below, these graphs are to _contextualise_ the headlines.
The thicker grey line indicating ***consumption*** is _significant_; it shows the point (in 2015) when Portugal became a **Net _Exporter_** of Electricity. 
For _all_ the stats/metrics and ratios of consumption, export and efficiency, see:
https://tradingeconomics.com/portugal/energy-imports-net-percent-of-energy-use-wb-data.html

## Headlines?

There is no shortage of headlines stating that Portugal generated over 100% of energy demand from renewables. e.g: Renewable energy generated 104% of Portugal's electricity consumption in March
https://www.independent.co.uk/news/world/europe/portugal-renewable-energy-generated-electricity-consumption-march-greenhouse-gas-environment-a8289656.html

But this headline is _deeply_ misleading. (_bordering on "fake news"..._)

## _Actual_ Energy Production: 

The annual breakdown of energy production is _far_ less than 100%:
https://www.apren.pt/contents/publicationsreportcarditems/12-renewable-electricity-in-portugal.pdf
![energy-breakdown-portugal-2017](https://user-images.githubusercontent.com/194400/47278615-53bbfa00-d5c2-11e8-9ddf-cb539fcb8138.png)

+ https://www.ren.pt/pt-PT/quem_somos/facts_and_figures/
+ https://www.apren.pt/en/renewable-energies/production
+ http://www.ren21.net/wp-content/uploads/2018/06/17-8652_GSR2018_FullReport_web_final_.pdf

## _Current_ Data

The _latest_ energy consumption stats are available online: https://www.ren.pt/en-GB 
for **September 2018 _only_ 33.8%** of energy consumed was from renewable sources:
![ren-production-breakdown-sept-2018](https://user-images.githubusercontent.com/194400/47279562-df855480-d5c9-11e8-955b-52340b88e1f6.png)

https://en.wikipedia.org/wiki/Renewable_energy_in_Portugal
![image](https://user-images.githubusercontent.com/194400/47278715-00967700-d5c3-11e8-9367-41db73cbbf89.png)

# Solar?

![solar-panels-on-rooftop](https://user-images.githubusercontent.com/194400/47258195-a5b03300-d48f-11e8-8eec-4829469aa179.png)

Solar Panels have always been _high_ on our list of features that are (_very_) "nice to have". ☀️🔋
We _budgeted_ for Solar in our _original_ plan: [/dwyl/phase-two/braga#financials](https://github.com/dwyl/phase-two/tree/817a0fe54e702d58394899edc8840ec70b159adf/braga#financials)
And if we can get a good quote (_including installation_) from reputable supplier, we will go for it!

Having Solar is _not_ a "blocker" to opening the house so have not been a _focus_;
We don't _need_ to be "energy independent" or spend €40k on panels, inverters and batteries for "**MVP**". 

We have "planned ahead" when doing the electrical re-wiring of the house and the cables are all in place to get the solar energy from the roof to the basement next to the mains consumer unit (_where inverters, charge controller and batteries will "live"_). 
When the time comes to install the panels and supporting equipment, we expect _minimal_ "disruption" to the house for scaffolding on the north wall and installation should be complete within a couple of weeks. 

# Todo 

+ [x] Prepare Roof (Drone) Video to send to suppliers with RFP (_detailed context of our requirements_)
+ [x] Prepare RFP to send to suppliers.
  + [x] Surface Area in m<sup>2</sup>
  + [x] Sun Exposure estimate
+ [x] Send RFP emails
  + [x] Follow up with phone call 2 hours after sending email.
+ [x] Get Quote for Tesla PowerWalls

# Solar Panel Suppliers (_in Portugal_) ☀️ 🇵🇹 

Our research indicates that Damia Solar are the best _value_ suppliers in PT:
https://www.damiasolar.com/produtos/kit_energia_solar/kit-de-autoconsumo-solar-4000w-de-injec-o-zero_da1002_112
![image](https://user-images.githubusercontent.com/194400/39959886-7f3ba1a8-5610-11e8-8b10-5b8b3be81031.png)

Most of the **kits** available include **OPZS** Batteries https://en.wikipedia.org/wiki/Opzs
this is a Lead Acid battery generally composed of diluted **sulfuric acid** https://en.wikipedia.org/wiki/Sulfuric_acid ... I think we will give it a miss.

# Is Solar Worth It?

On purely economic terms, solar energy is a _poor_ use of capital. 💸 
The _best_-case scenario "pay back period" for an investment in panels is 8 years.
The _reality_ is more like 12 years without government incentives.
Even if we _assume_ near-perfect panel positioning we can only expect around 5 hours 
of sunlight (_on average_) per day. 

## Average Sunshine Per Day

Thankfully there is decent historical data freely available online for sunshine hours:

https://www.worldweatheronline.com/braga-weather-averages/braga/pt.aspx
![image](https://user-images.githubusercontent.com/194400/47264560-3da23100-d511-11e8-8a8b-c049062b1c46.png)

The _lowest_ sunshine recorded (_in the available data_) is **147 hours** (_averaged over Jan - March_)
![image](https://user-images.githubusercontent.com/194400/47264612-1e57d380-d512-11e8-895d-f58b29083351.png)

Drilling down into the data we see that the lowest sunshine was **January 2016** with **94.5h**. 
![image](https://user-images.githubusercontent.com/194400/47264704-ae4a4d00-d513-11e8-995f-e4c95c3e92d5.png)

We need to account for this and plan to source our energy from the National Grid in January each year.

There are _two_ reasons for this dip: seasonal variation in angle of incidence of solar rays
and fewer days in the denominator. i.e. February typically has 28 days.

## A reality check on renewables - David MacKay

This TED-Ed talk gives a good overview of the _arithmetic_ for energy generation:

[![image](https://user-images.githubusercontent.com/194400/47258171-35091680-d48f-11e8-8c97-360945118b06.png)](https://youtu.be/E0W1ZZYIV8o)
https://youtu.be/E0W1ZZYIV8o

# Relevant Reading

A book I have found particularly useful in understanding solar is:
https://www.amazon.co.uk/Solar-Electricity-Handbook-Installing-Photovoltaic/dp/1907670688
![image](https://user-images.githubusercontent.com/194400/47264773-c1115180-d514-11e8-93e2-93f3fe75dc6c.png)

If you don't have the time to read a 186 page book, there are a few good sites you can look at:

+ [x] Basic Solar Math: https://livingonsolarpower.wordpress.com/2013/03/23/basic-solar-energy-math/
+ [x]  Solar energy generated m<sup>2</sup>: https://www.quora.com/How-much-solar-energy-is-generated-per-square-metre
+ [x] Solar Payback Period: https://news.energysage.com/understanding-your-solar-panel-payback-period note: this calculation assumes government financial "incentives"; we have none.
+ [x] Renewable Energy in Portugal:
http://solar-algarve.com/welcome-to-solar-algarve-for-solar-energy-solutions/grid-connected-feed-in-tariffs/

## Related Background Reading and General Knowledge

+ Warren Buffet doesn't _want_ you to have solar panels: https://www.fool.com/investing/2016/08/24/why-warren-buffett-wants-keep-solar-panels-off-you.aspx

# `#helpwanted`

If anyone has _experience_ of installing _domestic_ solar panels
and has any advice/wisdom to share, please comment below!

**Comment by nelsonic** [comment]
The above graphs are for _Electricity_ usage.  If we look at  **Total Primary Energy Supply** (TPES), the story is _even_ more skewed toward hydrocarbons!
https://www.iea.org/publications/freepublications/publication/Energy_Policies_of_IEA_Countries_Portugal_2016_Review.pdf
![total-primary-energy-supply](https://user-images.githubusercontent.com/194400/47284229-7fe57400-d5de-11e8-82d8-9976b9146150.png)

Portugal relative to other IEA member countries in 2014.
![total-energy-oecd-countries](https://user-images.githubusercontent.com/194400/47284405-2598e300-d5df-11e8-9342-aa709615cfa8.png)

Also relevant: https://en.wikipedia.org/wiki/Pumped-storage_hydroelectricity
(_energy storage capacity is not expanding at the rate of renewable supply ..._)

**Comment by nelsonic** [comment]
Consider adding ERoEI calculation to explain solar panels from a environmental perspective:
https://en.wikipedia.org/wiki/Energy_returned_on_energy_invested

**Comment by nelsonic** [comment]
Companies to request solar panel + installation quotes from:
+ Damaia: https://www.damiasolar.com/produtos/kit_energia_solar/kit-de-autoconsumo-solar-4000w-de-injec-o-zero_da1002_112
+ [https://www.merkasol.com](https://www.merkasol.com/epages/62387086.sf/pt_PT/?ViewAction=View&ObjectPath=/Shops/62387086/Categories/%22Kits+Solares+y+Eolicos%22/Conexion_Aislada/Kit_Solares_Fotovoltaicos&OrderBy=ListPrice&OrderDesc=1&PageSize=50&Page=1)
+ FF Solar: http://www.ffsolar.com/index.php?lang=PT&page=produtos&content=model&field=1&category=45&family=241
+ Solar Full: https://www.olx.pt/anuncio/kit-5-000w-habitao-painel-solar-fotovoltaico-pico-10kw-prod-2750w-IDCdyaa.html

**Comment by nelsonic** [comment]
Through research, we note that the SunPower® X-Series Residential Solar appear to be the _best_ panels. see: https://us.sunpower.com/sites/sunpower/files/sunpower-x-series-residential-solar-panels-x21-335-blk-x21-345-datasheet-504828-revg.pdf
![image](https://user-images.githubusercontent.com/194400/50150400-be816d00-02b5-11e9-817e-321ee242ae3c.png)

https://www.ebay.co.uk/itm/4kw-SUNPOWER-X21-335W-Black-Solar-PV-System-THE-WORLDS-MOST-EFFICIENT-PANEL/132884779246
![image](https://user-images.githubusercontent.com/194400/50150452-effa3880-02b5-11e9-8495-3e51bd577b58.png)

![image](https://user-images.githubusercontent.com/194400/50150696-a3632d00-02b6-11e9-9c1f-86128b103a8d.png)

**£5,995** for 12 panels (***4kW***) is _definitely_ not "_cheap_"!
But if our objective is to maximise the amount of energy we can generate from the available _space_,
then we should _only_ be considering panels that have this level of efficient conversion.

**Comment by nelsonic** [comment]
Sketchup Layout of Solar Panels:
![image](https://user-images.githubusercontent.com/194400/54815892-e0aad280-4c8a-11e9-9815-67d3af635c07.png)

**Comment by nelsonic** [comment]
@iteles we need to ask the suppliers of our solar panels what the price of the higher output panels are.

For reference, the SunPower X1 335W panels I linked to above (£5,995 for 12 panels) cost €620 _each_ from Portuguese importer/installer "FF Solar" see: http://www.ffsolar.com/index.php?lang=PT&page=produtos&content=model&field=1&category=1&family=3
![image](https://user-images.githubusercontent.com/194400/54847139-7c613080-4cd5-11e9-8d65-3f2781c28019.png)

http://www.ffsolar.com/products/FF_lista_PVP.pdf
![image](https://user-images.githubusercontent.com/194400/54847236-bfbb9f00-4cd5-11e9-8f9d-c01efca4c5f3.png)

I think that _sadly_, the SunPower X1 are cost-prohibitive considering they have only 10% more output than a _much_ more affordable 300W panel which costs around €160 ...

My original _estimate_ in the April 2017 Biz Plan was €40k for panels and storage. 
It's looking like with the recent price reductions that we might be able to get our complete solar system for €20k (_with installation!_) so that would be _awesome_! (_the first underspend on the project!_) 😮

**Comment by nelsonic** [comment]
@iteles I exported the Roof Dimensions with Windows and _Suggested_ Solar Panel positioning as PDF:
[braga-telhado-janelas-e-paineis-solares.pdf](https://github.com/dwyl/home/files/2998175/braga-telhado-janelas-e-paineis-solares.pdf)

This should be easier for Carlos (Solar Panel Engineer) to view. 🔍 
(_Please reply to his email with this as an attachment, thanks!_) 📎

**Comment by iteles** [comment]
We met with solar panel providers at `home` this morning and discussed the positioning of the solar panels and required wiring.

The salient points from this meeting were:
+ The panels can only be added to two of the sides of the roof
  + This is because all of the solar panels in one array must have a similar level of exposure to sunlight - i.e. the same angle and face the same way
+ There must be a minimum of 10 solar panels (of the ones that have been proposed) wired in series, so we could not 'make use' of the space we would currently have left on the right hand side of the roof if we were to keep the heat pump solar panels there (green in the drawing above)
+ The wiring will need to go down to where the inverter and batteries will be and also to the main electricity box
  + We have put in place a plan for this and it will have to be completed before the rest of the flooring goes down 

### Open questions
+ Would it be more advantageous to use the prime sun location (right hand side of the roof drawing above) for additional photovoltaic panels instead of the heat pump panels?
  +  For this, we need to know the Seasonal Coefficient of Performance ([SCOP](https://iqenergy.ddns.net/faq/9.html)) of the heat pumps (I have emailed to request this and they came back with the COP of the solar panels, so I have replied to explain this is not what we need)
+ Budget for higher capacity solar panels (330kW or 360kw) with the same surface area

**Comment by iteles** [comment]
🤦‍♀️ I had already started writing the above and the issue only updated with your latest comments when I submitted it @nelsonic 

That'll teach me for getting distracted with emailing people part-way through a comment!

I had already emailed them both the drawing and the request for the updated quote 👍

**Comment by nelsonic** [comment]
The cost of Photovoltaic solar has decreased by 18% in the last year ... 😮 
https://about.bnef.com/blog/battery-powers-latest-plunge-costs-threatens-coal-gas/

**Comment by iteles** [comment]
Chased today for the SCOP (to know whether we will need to by law have solar panels for the heat pumps specifically) and also for the spec of the solar panels for the heat pumps.

**Comment by iteles** [comment]
Have been chasing constantly but no joy.

In the meantime, now that we have a date for the roof #35, I've emailed the photovoltaics guys to see what we need to decide/put in place ahead of this getting done.

**Comment by iteles** [comment]
The latest proposals/costs for the solar panels include something interesting: solar panels on _three_ portions of the roof. **I have emailed them to query this**.

All quotes are not for **315W** panels.

They also include batteries for now - see note in https://github.com/ideaq/home/issues/25#issuecomment-495939542 around costs for the decision to be made.

![image](https://user-images.githubusercontent.com/4185328/58373049-3e73c900-7f1f-11e9-8f67-a221edaaff06.png)

![image](https://user-images.githubusercontent.com/4185328/58373194-336e6800-7f22-11e9-8f89-8ddcb8f8d424.png)

If only 2 sides on the roof:
![image](https://user-images.githubusercontent.com/4185328/58373198-45500b00-7f22-11e9-92dc-6de41aa4caf7.png)
![image](https://user-images.githubusercontent.com/4185328/58373207-644e9d00-7f22-11e9-9424-9e2e23d9d9f3.png)

**Comment by iteles** [comment]
@nelsonic These guys seem extremely knowledgeable and they're _up the road_ if anything goes wrong. Do you think we should get a quote from the Póvoa de Lanhoso guys too or are you happy with these?

**Comment by nelsonic** [comment]
@iteles as you say, the supplier that has their HQ nearer to @home is preferable.
I don't think it would _hurt_ to get a quote from the Povoa people to determine if they are competitive ... but from reading their marketing material last time we were there, their panels were more expensive ...

**Comment by iteles** [comment]
@nelsonic DAPE have now confirmed it will take 3 weeks to source all the materials needed so we should confirm with them this coming week (3-Jun-2019).
 
**The solar panels on 3 portions of the roof is confirmed** (they confirmed this _was_ possible: "colocação de 8 módulos a melhor situação será dividir estes módulos pelos módulos das outras duas águas de modo a termos 2 strings com tensão equivalente").

**The only decision left is whether or not to go for batteries.** All costs and differences are here:
https://github.com/ideaq/home/issues/25#issuecomment-495939542

**Comment by iteles** [comment]
Confirmed we will have only photovoltaics as the solar panels for the heat pumps would mean that any energy produced from these panels _after_ the water was hot would just be wasted. With photovoltaics, we can store this extra energy and it won't go to waste.

**Comment by iteles** [comment]
The technician from DAPE will be there at 09h00 tomorrow morning (Wednesday 3-Jul-2019) to discuss anything that needs to be done with the roofers.

**Comment by iteles** [comment]
There was a delay in the delivery of the batteries, which should arrive at the end of next week.

Installation of everything has been booked in starting on 22nd July 2019.

**Comment by nelsonic** [comment]
We are still waiting for the batteries and inverter to be installed before we can _use_ the panels ... ⏳ 
But the 32 panels have already been installed on the roof:
![braga-home-solar-panels](https://user-images.githubusercontent.com/194400/64314612-85a23900-cfa7-11e9-94ad-c1cd6a93c99e.jpg)

**Comment by iteles** [comment]
I spoke to the solar panel company yesterday and they are expecting the battery and inverters _this week_ (but they said that last week so I'm not holding my breath).

**Comment by iteles** [comment]
Batteries and inverters have now been dispatched from Germany and are expected 11-Sep-2019.
Installation this week or next at the latest.

**Comment by nelsonic** [comment]
@iteles means we need Bruno to "inject" the solar power into the mains so that we can _test_ it. 😉

**Comment by iteles** [comment]
Another delay. 
Email from the transporter (in German, thank you Google Translate):
I can assure you 100% that the goods were picked up on 04.09.2019 and on the way to Portugal. Unfortunately, the shipment did not arrive punctually and therefore missed the departure on Thursday (05.09.2019) to PT. Therefore, the goods were loaded with the next departure and is now on the way to PT. We very much regret the incident. **Delivery 18.09.2019.**

**Comment by iteles** [comment]
These are now installed _however_, when the solar panels were put on the roof, the guttering #67 was bent, so we discovered this week that ALL the water was collecting in the South corner of the house and the weight of the water was almost about to pull the guttering off the roof.  
It was also causing ALL of the water coming off the roof to rush down and into the garage, causing a small flood across the whole basement level.

The flooding issue has now been sorting by digging a path for this water towards the garden whilst we wait for a more permanent solution but the gutters will need to be fixed.

I have spoken to Carlos from DAPE (solar panels) and we'll see whether the solution will be to get Ze Manuel/Rafael out to fix them or whether DAPE will sort it out.

The solar system will also be connected to the electricity system this week.

**Comment by iteles** [comment]
This  has now been completely installed, both on the roof and hooked up to the electrical circuit (completely in early November 2019).
It _should_ already be working, producing electricity that can be used in the house.

There are two steps missing here:
+ **Final paperwork for EDP:** There needs to be paperwork submitted so that we can feed our extra electricity into the EDP network (the panels will be directed for home use first, then fill the batteries and only if there is spare electricity being produced will it be fed into the EDP network) - this can only be done _after_ our electricity supply has been increased https://github.com/dwyl/home/issues/70
+ **Internet**: Once we have the network up and running, they will come back and hook us up with the application so that we can see electricity production and flow.

**Comment by iteles** [comment]
We got our latest electricity bill which hadn’t changed and called them. 

All the equipment is hooked up but apparently they haven’t started it yet because they require WiFi to ensure that the flow of energy is correct and everything working properly, particularly before our power is increased. 

We agreed they would start everything up with a WiFi hotspot (obviously) and will let me know which day this will happen.

**Comment by nelsonic** [comment]
@iteles should we place the order with the ISP _today_ so that we can get this ball moving?
The amount we are paying each month to EDP to run our dehumidifier almost outweighs the cost of broadband! And we are going to need a _lot_ of electricity very soon to heat water which was the _whole point_ of getting the solar panels ... 

The conclusion of our research last year is still the same: MEO Fibra **1GB €38,99/mês** (_the fastest_)
https://www.meo.pt/pacotes/fibra/outros-pacotes
![meo-fibra-1gb](https://user-images.githubusercontent.com/194400/71910173-5ecc5d00-3169-11ea-9050-ed4acf684e13.png)

Can I place the order _today_?

**Comment by iteles** [comment]
The question above was moved to #107.

The plan is for the solar panel folks to come today (20-Jan-2020) and get them up and running.

**Comment by nelsonic** [comment]
I'm here waiting for the Solar Panel people to arrive. ⏳ 
Hot spot indeed. Who is _providing_ it? ❓ 
(_as much as I want to donate my data to use the solar panels, 
I'm 100% certain the software will require an update as soon as it connects to the net..._) 🙄

**Comment by iteles** [comment]
Now that the [internet has been hooked up](https://github.com/dwyl/home/issues/107#issuecomment-578099725), I've called the solar panel folks to come back and finalise the installation of the application. This is provisionally expected to be on Monday/Tuesday 27/28-Jan-2020.

**Comment by nelsonic** [comment]
Screenshot of Solax UI showing some lovely solar rays powering @home ☀️
![Screenshot_20200124-174132~2](https://user-images.githubusercontent.com/194400/73090919-3c5c7280-3ed1-11ea-86ab-ecad348ca8d0.png)

**Comment by iteles** [comment]
We received confirmation on 27-Jan-2020, after some testing on our part, that there is a bug with the 'medidor' software so it is not measuring and therefore not dispensing the correct amount of energy to meet the (currently low) household needs.

Following up on this next week.

**Comment by nelsonic** [comment]
Indeed the solar output measurement software has a bug in it. Not very reassuring ... 🙄

**Comment by iteles** [comment]
@nelsonic The solar panel folks will be here this afternoon sometime after 15h00 to attempt to fix the issues.

**Comment by nelsonic** [comment]
@iteles thanks for following up. 👍 📆

**Comment by iteles** [comment]
Our solar panels have been working properly since late March 👍 

We're now around the height of Summer:
![image](https://user-images.githubusercontent.com/4185328/87780512-c9e8a180-c826-11ea-89fc-80692fbff09c.png)

(Those minimums are a joke, I _wish_ it cooled down overnight but at 23h00 it's still 30 degrees out!)

And our outputs are looking pretty good!

Just after 11am:

![IMG_2102](https://user-images.githubusercontent.com/4185328/87780557-dec53500-c826-11ea-9a65-74ed55a49b1d.PNG)

And peak production around midday:

![IMG_2105](https://user-images.githubusercontent.com/4185328/87780562-dff66200-c826-11ea-85d7-d17e7b78c647.PNG)

**Comment by iteles** [comment]
We have now agree to the new contract from EDP, so I am pulling together the information required by DAPE for them to finalize their configuration:
+ CPE number
+ NIF
+ Potência contratada

**Comment by iteles** [comment]
This info has now been sent and they are putting in the request for the registration of the solar power system with EDP (as is mandated by law).
I have warned them that the new potência contratada #70 won't be in force until early September but they have said they want to put the request in now as it actually takes quite a long time.

@nelsonic They have also said that we need to have the 'contador' hooked up by law, which requires a SIM card with data capabilities to be installed in the GSM modem that is already installed downstairs.
Carlos mentioned they could provide this for €5 pcm but that most people prefer to get it through their existing providers as they get them at €2 or €2.50 pcm.

This is the first I remember hearing of this so I have requested the spec for the equipment:
[Contador Itron ACE6000.pdf](https://github.com/dwyl/home/files/5097512/Contador.Itron.ACE6000.pdf)
[Modem GSM ITRON.pdf](https://github.com/dwyl/home/files/5097511/Modem.GSM.ITRON.pdf)

I suspect that this equipment is purely for the sale of electricity to EDP. If so, we will be spending more money on the data card every month than we'd get back in sales at this stage.
So I've asked the question on whether it's necessary and if so, how much data is required.

**Comment by nelsonic** [comment]
2 years later and Google Earth still thinks we have the old roof ... 🤷‍♂️ 

![google-earth-no-solar-panels-anon](https://user-images.githubusercontent.com/194400/157443272-947aef3c-bbfe-44de-b8c5-c7257bd5118c.png)
 
Need to dig out a drone photo. 📷

**Comment by iteles** [comment]
These have been working since. Issues to be opened on maintenance at this point I reckon!

**Comment by nelsonic** [comment]
@iteles Yeah I was thinking we could pass a pipe/hose with water from the Well #79 up to the roof _through_ the caleira
and put a Tap 🚰 on the roof which we can then connect a hose to and spray (pressure wash) the panels periodically. 💭
